website/settings/local-dist.py

```diff
@@ -22,6 +22,11 @@
 #WATERBUTLER_URL = 'http://localhost:7777'
 #WATERBUTLER_INTERNAL_URL = WATERBUTLER_URL
 
+LIVE_RELOAD_DOMAIN = 'http://localhost:4200'
```

Comment:
The live reload asset from the root path can only be served from one place at a time. It's possible to run ember serve on a different port then adjust the port here in local.py. This is overridden by the environment variable with Docker anyhow.